doc-Appliance_Hardening_Guide/cfme/master.adoc

```diff
@@ -0,0 +1,30 @@
+= Appliance Hardening Guide
+:vernum: 4.1
```

Comment:
Use the reserved `{product-version}` parameter here, whose value comes from each branch's `name` value in the `_distro_map.yml` file (see [example](https://github.com/openshift/openshift-docs/blob/master/_distro_map.yml#L36))